### ISSUE TITLE
Collection with formdata malformed for API

### ISSUE THREAD

**Issue by KriKrus** [knuckleswtf/scribe#881]
### Scribe version

4.37.1

### PHP version

8.3.9

### Framework

Laravel

### Framework version

11.20.0

### Scribe config

```ruby
title => "xxx API Dokumentation"
auth.enabled => true
auth.default => true
auth.name => "Authorization"
logo => "logo.svg"
examples.faker_seed => 1234
strategies.headers => added [
    'override',
    [
        'Content-Type' => 'application/json',
        'Accept' => 'application/json',
        'im-client-type' => 'app',
        'im-app-version' => '2024.08.14',
        'im-device-id' => 'id123',
    ],
]: removed [
    'override',
    [
        'Content-Type' => 'application/json',
        'Accept' => 'application/json',
    ],
]
```

### What happened?

The Command generates the postman collection and I want to upload it to the postman api using the [put api](https://documenter.getpostman.com/view/6034746/SzYUZg9Q), but the request gets rejected because of a validation failure.

The problem is following json part in a formdata request: 
``` 
{
    "key": "size",
    "value": 84,
    "type": "text",
    "description": ":Attribute muss mindestens 0 sein."
}
```
which is generated from following rules of a form request:
```

    public function rules(): array
    {
        return [
            'size' => ['required', 'numeric', 'min:0'],
        ];
    }
```

Postman rejects the file, because the form data is set to type "text", but the value is an integer. Can the generation of the collection respect the type? I think the change needs to be made in the PostmanCollectionWriter in getFormDataParams, if its not type file always cast the value to a string?

### Docs

- [X] I've checked [the docs](https://scribe.knuckles.wtf/laravel), the [troubleshooting guide](https://scribe.knuckles.wtf/laravel/troubleshooting), and [existing issues](https://github.com/knuckleswtf/scribe/issues?q=is%3Aissue+), but I didn't find a solution

**Comment by shalvah** [comment]
Good catch. Could you send in a PR to fix?

**Comment by KriKrus** [comment]
Hey,
sorry for the late response. I was on vacation and had a lot todos. Yes, i can try to send a PR in the following weeks.

**Comment by KriKrus** [comment]
Sorry it took so long, but a lot other things had to be done. But I created the PR.

**Comment by shalvah** [comment]
Released in 4.39.0
